deploy/charts/external-secrets/templates/rbac.yaml

```diff
@@ -82,6 +82,16 @@ rules:
     - "configmaps"
     verbs:
     - "create"
+{{- if .Values.leaderElect -}}
+  - apiGroups:
+    - "coordination.k8s.io"
+    resources:
+    - "leases"
+    verbs:
+    - "get"
+    - "update"
+    - "patch"
+{{- end -}}
```

Comment:
The chart check is failing because both `if` and `end` should be `{{- end }}` without the trailing newline removal.